.github/labeler.yml

```diff
@@ -1,5 +1,74 @@
 # Add labels to any any pull request with changes to the specified paths
-
+# for reference, see https://github.com/actions/labeler
 
 github:
 - .github/**/*
+
+documentation:
+- CONTRIBUTING.md
+- LICENSE
+- README.md
+- backend/README.md
+- doc/how-to-guides/develop-with-neo4j.md
+- doc/how-to-guides/docker-compose-setup.md
+- doc/introduction/setup-dev.md
+- doc/references/taxonomy-parser.md
+- doc/assets/.empty
```

Comment:
```suggestion
- CONTRIBUTING.md
- LICENSE
- README.md
- backend/README.md
- doc/**/*
```

Reply:
I added - doc/**/* but I want a list of important files we can point people to ( code pointers)